CMakeLists.txt

```diff
@@ -274,6 +276,12 @@ if (VEXCL_MASTER_PROJECT)
         COPYONLY
         )
 
+    configure_file(
+        "${CMAKE_CURRENT_SOURCE_DIR}/cmake/VexCLBoostTargets.cmake"
+        "${CMAKE_CURRENT_BINARY_DIR}/cmake/VexCLBoostTargets.cmake"
+        COPYONLY
+        )
```

Comment:
I think had problems with `.cmake` having same name both in source and binary dirs (cmake tried to pick up the on in source dir). Would it be better to rename the one in source to `.cmake.in`?

Reply:
`.in` should only be used if you are changing the file. Sounds like a path issue somewhere, I'll check. Really should not have to copy.

Reply:
Ah, I see why the copy is done. I think just making sure the paths are correct should be.